client/state/sites/features/reducer.js

```diff
@@ -37,7 +37,7 @@ function updateBulkFeatures( state, features ) {
 		newState[ siteId ] = {
 			...initialSiteState,
 			hasLoadedFromServer: true,
-			data: features,
+			data: features[ siteId ],
```

Comment:
This was a bug in the bulk features requests; it wasn't saving correctly for the sites, so it wouldn't load the options to update/install a plugin on "Manage sites" CTA.

Reply:
Nice catch!!